server/game/cards/19.1-AS01/Neutral/MangroveSafehouse.js

```diff
@@ -0,0 +1,90 @@
+const AbilityDsl = require('../../../abilitydsl.js');
+const {
+    CardTypes,
+    Players,
+    Locations,
+    Elements
+} = require('../../../Constants.js');
+const DrawCard = require('../../../drawcard.js');
+
+const elementKey = 'mangrove-safehouse-water';
+
+class MangroveSafehouse extends DrawCard {
+    setupCardAbilities() {
+        this.persistentEffect({
+            targetLocation: Locations.Provinces,
+            targetController: Players.Self,
+            condition: (context) => {
+                const currentRingToClaim =
+                    this.getCurrentElementSymbol(elementKey);
+                const youHaveNotClaimedTheRequiredRing = !this.game.rings[
+                    currentRingToClaim
+                ].isConsideredClaimed(context.player);
+                return youHaveNotClaimedTheRequiredRing;
+            },
+            match: (card, context) => {
+                const isAdjacentProvince = context.player.areLocationsAdjacent(
+                    context.source.location,
+                    card.location
+                );
+                return isAdjacentProvince;
+            },
+            effect: AbilityDsl.effects.modifyProvinceStrength(-1)
+        });
+
+        this.action({
+            title: 'Move an attacker out of the conflict',
+            effect: 'move {0} home {1}',
+            effectArgs: (context) => [
+                this.targetIsMantis(context) &&
+                this.opponentHasFateToBeStolen(context)
+                    ? 'and steal 1 fate'
+                    : ''
```

Comment:
The space should be before the `and` in `and steal 1 fate`
ex:
```
'move {0} home{1}'
' and steal 1 fate'
```

Or you'll get a message `to move SoandSo home ` with a space at the end